articles/azure/azs-how-create-vmfromsnapshot-portal.md

```diff
@@ -0,0 +1,147 @@
+
+### Intended audience
+
+To complete the steps in this article, you must have appropriate access to a subscription in the Azure Stack Hub portal.
+
+## Creating a snapshot from a disk
+
+1. Log in to the Azure Stack Hub portal.
+
+    For more detailed instructions, see the [*Getting Started Guide for UKCloud for Microsoft Azure*](azs-gs.md).
+
+2. Click **All services** in the favourites panel, then select **Disks** under the *Compute* section.
+
+    ![All services - disks](images/azs-browser-allservices-disks.png)
+
+3. From the list, select the disk that you would like to create a snapshot from.
+
+    ![Disks blade](images/azs-browser-disks-list.png)
+
+    > [!NOTE]
+    > Snapshots can only be created from managed disks.
+
+4. In the new blade for the selected disk, click **Create snapshot**.
+
+    ![Create snapshot from disk - New Blade](images/azs-create-snapshot-disk.png)
+
+5. In the **Create snapshot** blade, enter the following information:
+
+    - **Name** - The name of the snapshot.
+
+    - **Subscription** - This will be your UKCloud for Microsoft Azure subscription.
+
+    - **Resource group** - Select an existing resource group, or create a new one by typing a name for your new resource group.
+
+    - **Location** - This will be the location of the Azure Stack Hub.
+
+    - **Account type** - Choose either Standard (HDD) or Premium (SSD) storage.
+
+    ![Create snapshot from disk - Populate & Create](images/azs-browser-create-snapshot.png)
+
+6. When you're done, click the **Create** button to create the snapshot.
+
+7. You can monitor the progress of the snapshot creation by clicking the **Notifications** icon.
+
+    ![Notification showing snapshot creation in progress](images/azs-browser-create-snapshot-progress.png)
+
+## Creating a new managed disk from a snapshot
+
+1. Click **All services** in the favourites panel, then select **Disks** under the *Compute* section.
+
+    ![All services - disks](images/azs-browser-allservices-disks.png)
+
+2. On the **Disks** blade, click **Add**.
+
+    ![Disks blade - Add](images/azs-browser-disks-list-add.png)
+
+3. In the **Create managed disk** blade, enter the following information:
+
+    - **Name** - The name of the disk.
+
+    - **Subscription** - This will be your UKCloud for Microsoft Azure subscription.
+
+    - **Resource group** - Select an existing resource group, or create a new one by typing a name for your new resource group.
+
+    - **Location** - This will be the location of the Azure Stack Hub.
+
+    - **Account type** - Choose either Standard (HDD) or Premium (SSD) storage.
+
+    - **Source type** - Select *Snapshot* from the dropdown.
+        - **Source snapshot** - Select the previously created snapshot.
+
+    - **Size** - This will be set to the size of the source disk that the snapshot was created from. You can increase this if necessary, but you will not be allowed to decrease it below the original size.
```

Comment:
Maybe the part about not being able to decrease it should be in a !NOTE or !WARNING. To draw more attention to it?

Reply:
Updated in latest commit.